### PR TITLE
5x5 sse2 kernel and new interface support

### PR DESCRIPTION
**Pull request by harouwu**
ninja convolution-test
ninja q8mpdw-test
ninja q8updw-test
benchmark working

### CODE REVIEW

File: src/init.c [hunk 1]
```diff
@@ -88,6 +88,10 @@ static void init(void) {
       .updw = q8updw_ukernel_9c8__neon,
       .cr = 8,
   };
+  qnnp_params.q8dw25 = (struct q8mpdw_parameters) {
```
[the remark this answers is not in the document]
Comment (Maratyszcza) [reply src/init.c]: yes, nearly all new phones (e.g. Galaxy S6+) support ARM64

File: src/q8mpdw/25c8-sse2.c [hunk 2]
```diff
@@ -0,0 +1,742 @@
...
+      const __m128i vi23 = _mm_loadl_epi64((const __m128i*) i23); i23 += 8;
+      const __m128i vxi23 = _mm_sub_epi16(_mm_unpacklo_epi8(vi23, vzero), vinput_zero_point);
+      const __m128i vk23 = _mm_loadl_epi64((const __m128i*) ((uintptr_t) w + 104));
+      const __m128i vxk23 = _mm_sub_epi16(_mm_unpacklo_epi8(vk23, vzero), vkernel_zero_point);
+      const __m128i vprod23_odd  = _mm_mullo_epi16(vxi23, vxk23);
+      const __m128i vprod23_even = _mm_mulhi_epi16(vxi23, vxk23);
+      vacc_lo = _mm_add_epi32(vacc_lo, _mm_unpacklo_epi16(vprod23_odd, vprod23_even));
+      vacc_hi = _mm_add_epi32(vacc_hi, _mm_unpackhi_epi16(vprod23_odd, vprod23_even));
+
+      w = (void*) ((uintptr_t) w + 112);
```
Comment (Maratyszcza) [inline src/q8mpdw/25c8-sse2.c]: cast to `const void*`

File: src/q8mpdw/25c8-sse2.c [hunk 3]
```diff
@@ -0,0 +1,742 @@
...
+    size_t output_width,
+    const uint8_t** input,
+    const void* weights,
+    int32_t* outacc32,
+    uint8_t* output,
+    size_t input_stride,
+    size_t output_increment,
+    const union qnnp_conv_quantization_params quantization_params[restrict static 1])
+{
+  const __m128i vinput_zero_point = _mm_load_si128((const __m128i*) quantization_params->sse2.input_zero_point);
```
[the remark this answers is not in the document]
Comment (Maratyszcza) [reply src/q8mpdw/25c8-sse2.c]: nevermind, I'm being stupid here, your code is fine

**Comment by harouwu** [comment]
Comments addressed. Speed bench results(7-8 Gitem/s) are similar to 3x3dwconv sse2 on my Macbook with clang. merging now

**Pull request merged**